src/vendor.ts

```diff
@@ -28,9 +28,19 @@ import {
 import * as yaml from 'yaml'
 import * as _fs from 'fs-extra'
 import type { fetch } from 'node-fetch-native'
+import { AbortController } from 'node-abort-controller'
 
 export { exec, buildCmd } from 'zurk/spawn'
 
+import _createRequire from 'create-require'
+import { URL } from 'url'
```

Comment:
```ts
import { URL } from 'node:url'
```

Reply:
Thanks for highlighting.
https://github.com/google/zx/pull/794/commits/7114097514b4e2cf44df19f0690e6e39b9fc9bdf